index.js

```diff
@@ -11,6 +11,8 @@
  * @property {string|undefined} [language='en']
  *   Natural language of document.
  *   Should be a [BCP 47](https://tools.ietf.org/html/bcp47) language tag.
+ * @property {'ltr'|'rtl'|'auto'|undefined} [dir='ltr']
```

Comment:
```suggestion
 * @property {'ltr'|'rtl'|'auto'|undefined} [dir]
```

Defaulting to undefined seems fine here?